### DISCUSSION TITLE
thoughts on ids of spammers, channel owner name, moderator mode functionality

### DISCUSSION THREAD

**Discussion by Kurdakov** [ThioJoe/YT-Spammer-Purge#174] [Ideas]
When searching by user name their ids could be compared with author id ( some spammers tend to take author nicknames offering chat in WhatsApp to users etc - quite toxic spammers) so it would be important to catch them.

But for moderator mode it will require field - for which channel moderator has rights ( because without that it will be difficult to perform that search), 

the same goes for 5 recent videos - moderators and users could have field of channel in settings for which they intend to search, so they also could be searching 5 recent videos, as site owners.

p.s. for my needs I consider current utility feature complete. But maybe.

**Comment by ThioJoe** [comment]
Not sure what you mean, it already checks for obfuscated versions of the uploader's channel name, no matter who's scanning. Do you mean something else?

**Comment by Kurdakov** [comment]
I'd better spend more time with your code and if anything, I could add pull request later
